test/liquidation/makeLiquidatableProtocol.ts

```diff
@@ -54,75 +54,75 @@ export default async function makeLiquidatableProtocol() {
     extensionDelegate: extensionDelegate.address,
     baseToken: USDC,
     baseTokenPriceFeed: USDC_USD_PRICE_FEED,
-    supplyKink: 800000000000000000n, // .8e18
+    supplyKink: exp(0.8, 18),
     supplyPerYearInterestRateBase: 0n,
-    supplyPerYearInterestRateSlopeLow: 50000000000000000n, // .05e18
-    supplyPerYearInterestRateSlopeHigh: 2000000000000000000n, // 2e18
-    borrowKink: 800000000000000000n, // .8e18
-    borrowPerYearInterestRateBase: 5000000000000000n, // .005e18
-    borrowPerYearInterestRateSlopeLow: 100000000000000000n, // .1e18
-    borrowPerYearInterestRateSlopeHigh: 3000000000000000000n, // 3e18
-    storeFrontPriceFactor: 1000000000000000000n,
-    trackingIndexScale: 1000000000000000n,
-    baseTrackingSupplySpeed: 1000000000000000n,
-    baseTrackingBorrowSpeed: 1000000000000000n,
-    baseMinForRewards: 1000000n,
-    baseBorrowMin: 1000000n,
-    targetReserves: 1000000000000000000n,
+    supplyPerYearInterestRateSlopeLow: exp(0.5, 18),
+    supplyPerYearInterestRateSlopeHigh: exp(2, 18),
+    borrowKink: exp(0.8, 18),
+    borrowPerYearInterestRateBase: exp(5, 15),
```

Comment:
`exp(.005, 18)` for consistency?

Reply:
same with `exp(1, 15)` below. `exp(.001, 18)` instead?

Reply:
agree with first, for indices 15 is better, used in a lot of other tests this way